cypress/support/validations/index.js

```diff
@@ -19,4 +19,3 @@ import './miscellaneousValidation';
 import './decodeValidation';
 import './regExValidation';
 import './schemaValidation';
```

Comment:
was there no other way to fix this issue? Asking because this function was moved out of main.js as part of below ticket 
https://ccp.sys.comcast.net/browse/SYSTEST-9724

Reply:
https://cim.slack.com/archives/C05AJDEJ2DU/p1713873689785879
if you can go through this convo, snehal raised this issue and divya made same fix, but some how this was missed in open source FCS, I just added the change what divya made, pr link is present in that conversation.